config/locales/client.en.yml

```
@@ -2465,6 +2465,7 @@ en:
         success: "We've invited that user to participate in this message."
         success_group: "We've invited that group to participate in this message."
         error: "Sorry, there was an error inviting that user."
+        not_allowed: "Sorry, this user can't be invited."
```

```suggestion
        not_allowed: "Sorry, that user can't be invited."
```
(consistent with other translations above)